app/uk/gov/hmrc/agentaccesscontrol/service/AuthorisationService.scala

```diff
@@ -51,6 +51,16 @@ class AuthorisationService(desAuthorisationService: DesAuthorisationService,
         Future successful notAuthorised("No user is logged in")
     }
 
+  def isAuthorisedForPaye(agentCode: AgentCode, empRef: EmpRef)
+    (implicit ec: ExecutionContext, hc: HeaderCarrier, request: Request[Any]): Future[Boolean] =
+    authConnector.currentAuthDetails().flatMap {
```

Comment:
This will make 2 HTTP requests, one of which (enrolments) is not needed for PAYE. Is it like this to keep the code simple?